### PR TITLE
Sanitize - Implementation and tests

### PR DESCRIPTION
**Pull request by pangea-andrest**
- Missing docs and examples. Will be done once services is about to release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Replaced usage of the deprecated io/ioutil package.
 - Audit /download_results endpoint support
+- Sanitize service support
```
Comment (kenany) [inline CHANGELOG.md:14]: Should perhaps be under "Added".

**Review by kenany — DISMISSED** [review]

**Comment by pangea-andrest** [comment]
Change base to `beta`

**Comment by kenany** [comment]
Merged to `beta` ad 6698305b4a9a014fefc2ac6a868698bd70113069 and 79a27b45bb1cce68c47931c560ad5e2b7dad612a.

**Comment by pangea-andrest** [comment]
Already merged

**Pull request closed without merging**